docs/migration-guide/to-16.md

```diff
@@ -0,0 +1,13 @@
+# Migrating to 16.0.0
+
+This release contains breaking changes.
+
+## Breaking changes
+
+This breaking change may affect you:
+
+- removed deprecated stylistic rules
+
+## Removed Deprecated Stylistic Rules
+
+We've removed the stylistic rules we deprecated in 15.0.0. See the [15.0.0 migration guide](./to-15.md) for details of how to migrate.
```

Comment:
minor

> for details of how to migrate

Maybe that's a US thing, but I would prefer _for details on how to migrate_.